x/gov/handler.go

```diff
@@ -20,6 +20,7 @@ func NewHandler(keeper Keeper) sdk.Handler {
 		case MsgVote:
 			return handleMsgVote(ctx, keeper, msg)
 		case MsgSideChainDeposit:
+			// TODO: refund the token to the depositor after second hard fork
```

Comment:
No need for this TODO, because for all gov proposal, it will end within 2 weeks, so after two weeks all funds will go back to user's wallet. 

No need to do it by ourselves.